docs/release.rst

```diff
@@ -27,6 +27,8 @@ Fixes
 ~~~~~
 * Fixes issue with ``Delta`` Zarr 3 codec not working with ``astype``.
   By :user:`Norman Rzepka <normanrz>`, :issue:`664`
+* Cleanup ``PCodec`` soft dependency.
```

Comment:
```suggestion
* Cleanup ``PCodec`` soft dependency.
  Previously importing ``numcodecs.pcodec`` would work if ``pcodec`` is not installed,
  but now it will fail to import. This mirrors the behaviour of other optional dependencies.
```

Reply:
Would be good to have something that describes what's changed to users - I've left a suggesiton above